APIScrape/README.md

```diff
@@ -0,0 +1,26 @@
+TheCatApi // Developer Experience
+https://docs.thecatapi.com/
+
+
+## Authentication
+
+**Request Header**
+* The best & most secure way to send it
+* Set your API Key as the x-api-key header on every request.
+*e.g headers[“x-api-key”] = "ABC123"*
+
+Api key `ec254e44-3996-458b-8522-4933954d8fcd`
+
+
+**/breeds**
+
+GET
+https://api.thecatapi.com/v1/breeds/ populates select dropdown with all breeds.
+
```

Comment:
should detail that return contains a range of data for each breed including id (as will be used in image search) and name of each breed (as will be used in front end dropdown